pkgs/tools/system/snooze/default.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, fetchFromGitHub }:
+stdenv.mkDerivation rec {
+  pname = "snooze";
+  version = "0.4";
+  src = fetchFromGitHub {
+    owner = "leahneukirchen";
+    repo = "snooze";
+    rev = "v0.4";
```

Comment:
```suggestion
    rev = "v${version}";
```

Reply:
Thanks, applied.

Reply:
@sikmir Something else needs to be done before merging?

Reply:
LGTM